src/test/java/org/elasticsearch/discovery/zen/ping/unicast/UnicastZenPingTests.java

```diff
@@ -44,6 +45,7 @@
 /**
  *
  */
+@Slow
```

Comment:
I made this test slower due slow connection setup on the vagrant boxes we were using. I think we can speed it up again. I'll do this as a separate change